### PR TITLE
Add copy length warning

### PR DESCRIPTION
**Pull request by TomPretty**
## What does this change?
Add some UI to warn a user when creating a banner test with too much copy (causing the banner to overflow at certain breakpoints). 

**TODO**
- [x] update with proper limits
- [x] update copy?

## Images
Screenshot 2021-05-27 at 13 30 02

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,18 +21,19 @@
     "webpack-dev-server": "^3.8.1"
   },
   "dependencies": {
+    "@emotion/core": "^10.0.28",
     "@material-ui/core": "^4.11.0",
     "@material-ui/icons": "^4.5.1",
+    "@material-ui/lab": "^4.0.0-alpha.58",
     "@types/react-beautiful-dnd": "^13.0.0",
+    "emotion": "^10.0.27",
+    "emotion-theming": "^10.0.27",
```
Comment (TomPretty) [inline package.json:30]: There's a bit of noise here (I guess running npm install will re-organize everything?) - but all that's actually been added is the `@material-ui/lab` package to get the alert component

**Review by tomrf1 — APPROVED** [review]
great!
